internal/governor/conn.go

```diff
@@ -56,6 +67,7 @@ func Start(ctx context.Context, gitDir string) (*Conn, error) {
 			sock.Close()
 			return nil, err
 		default:
+			sideband.Write([]byte("continue\n"))
```

Comment:
Can you add a comment about why spokes-receive-pack needs to send "continue" here? (I assume it's because this is the code path that we hit when we fail open, for example when there's an error while connecting to governor?)